.vscode/settings.json

```
@@ -1,20 +1,36 @@
-	"prettier.trailingComma": "none",
-	"prettier.singleQuote": true,
 	"editor.formatOnSave": true,
+	"editor.defaultFormatter": "esbenp.prettier-vscode",
+	"editor.codeActionsOnSave": {
+		"source.fixAll": "explicit"
+	},
+	"prettier.requireConfig": true,
+	"prettier.configPath": ".prettierrc",
+	"prettier.useEditorConfig": true,
 	"eslint.format.enable": true,
 	"editor.tabSize": 4,
```

Biome (2.4.6)

[error] The key editor.tabSize was already declared.

(lint/suspicious/noDuplicateObjectKeys)